docs/features/pixel-art-converter.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Pixel Art Converter
+hide:
+  # - navigation
+  # - toc
+---
+
+The Pixel Art Converter tool aims to make it easier to show pixelart on a LED matrix panel, by converting any image to 2D pixelart and send it to the WLED device.
+
+## Installation approaches
+
+There are two ways to install the pixel art converter:
```

Comment:
You can also upload PixelArtConverter files using /edit, but this may have issues on ESP8266.